sample/index.php

```diff
@@ -29,10 +29,11 @@
     <div class="container">
         <div class="row">
             <div class="col-md-12">
-                <form class="form-horizontal form-main" method="GET">
-                      <input type="text" class="form-control" name="url" placeholder="Enter the URL from where images are to be extracted" required>
-                      <br>
-                      <button type="submit" class="btn btn-lg btn-success">Extract</button>
+                <form id="form-extractor" class="form-horizontal form-main" method="GET">
+                    <input type="text" class="form-control" name="url" placeholder="Enter the URL from where images are to be extracted" required>
+                    <br>
+                    <button type="submit" class="btn btn-lg btn-success">Extract</button>
+                    <div id="download-file"></div>
```

Comment:
You could have the button HTML here only, which is initially hidden and after the results are loaded, button could be shown